### PR TITLE
feat(ui): Status panel labels

### LINKED ISSUE #5418: Add labels to App Details Status Panel
To help both Argo CD beginners and screen readers, I think some small labels in the status panel would be helpful. This also complies better with [WCAG section 2.4.6](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=246).

Here's a quick mockup of how I think this should look: 

Screen Shot 2021-02-04 at 4 03 37 PM

### PR DESCRIPTION
**Pull request by rbreeze**
Fixes #5418, #5409

https://user-images.githubusercontent.com/6250584/107271971-dcfcd200-6a01-11eb-8c15-9f2f8de60aba.mov

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

File: ui/src/app/applications/components/application-details/application-details.tsx [hunk 1]
```diff
@@ -443,49 +443,49 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{nam
...
-                                        <SlidingPanel isShown={!!this.state.metadata} isMiddle={true} onClose={() => this.setState({metadata: null})}>
-                                            {this.state.metadata && (
-                                                <div className='white-box' style={{marginTop: '1.5em'}}>
-                                                    <div className='white-box__details'>
-                                                        <div className='row white-box__details-row'>
-                                                            <div className='columns small-3'>SHA:</div>
-                                                            <div className='columns small-9'>
-                                                                <Revision repoUrl={application.spec.source.repoURL} revision={this.state.metadata.revision} />
+                                        <SlidingPanel isShown={!!this.state.revision} isMiddle={true} onClose={() => this.setState({revision: null})}>
+                                            <DataLoader load={() => services.applications.revisionMetadata(application.metadata.name, this.state.revision)}>
```
Comment (alexmt) [inline ui/src/app/applications/components/application-details/application-details.tsx:447]: SlidingPanel content is rendered even when it is not shown. We need to add `this.state.revision && ..`. E.g.:

```
{this.state.revision && (
  <DataLoader load={() => services.applications.revisionMetadata(application.metadata.name, this.state.revision)}>
```

**Review by alexmt — CHANGES_REQUESTED** [review]
Looks great. Added one minor comment

**Review by alexmt — CHANGES_REQUESTED** [review]
Just noticed this breaks "sync windows" rendering:

Screen Shot 2021-02-09 at 5 06 09 PM

You can run command below to config sync window for default project:

```
kubectl apply -n argocd -f - << EOF                                                                                                                                                (⎈ |k3d-k3s-default:argocd-e2e)
apiVersion: argoproj.io/v1alpha1
kind: AppProject
metadata:
  name: default
spec:
  clusterResourceWhitelist:
  - group: '*'
    kind: '*'
  destinations:
  - namespace: '*'
    server: '*'
  sourceRepos:
  - '*'
  syncWindows:
  - applications:
    - '*'
    duration: 1h
    kind: allow
    schedule: 0 * 1 1 0
EOF
```

**Comment by rbreeze** [comment]
Ah good catch, thanks! Simple fix, forgot to remove old `columns` classname

**Review by alexmt — APPROVED** [review]
LGTM

**Pull request merged**